src/main/resources/templates/index.html

```diff
@@ -6,35 +6,35 @@
     <title>FindMyRecycling</title>
 </head>
 <body>
+    <div class="wrapper">
+        <header class="header">
+            <h2>FindMyRecycling</h2>
+        </header>
 
-    <div class="header">
-    <p>FindMyRecycling</p>
-    </div>
+        <nav>
+            <ul class="nav">
+                <li><a href="index.html">Search</a></li>
+                <li><a href="add-facility.html">Add Faculty</a></li>
+            </ul>
+        </nav>
 
-    <ul class="nav">
-        <li><a href="index.html">Search</a></li>
-        <li><a href="AddFacility.html">Add Faculty</a></li>
-    </ul>
+        <div class="box">
+            <p>Search for a recycling facility center near you!</p>
 
-    <div class="box">
-        <p>Search for a recycling facility center near you!</p>
+            <label for="recycleMaterial"></label>
+            <input type="text" id="recycleMaterial" name="recyclematerial" placeholder="Recyclable Material" >
```

Comment:
label and placeholder are redundant.  I'd stick with placeholder.  Less screen real estate used == less clutter, and a simpler UI.